package.json

```diff
@@ -24,16 +24,16 @@
     "scss/index.js"
   ],
   "engines": {
-    "node": ">=6"
+    "node": ">=8"
   },
   "peerDependencies": {
-    "stylelint": "^9.0.0 || ^10.0.0"
+    "stylelint": "^10.1.0 || ^11.0.0"
```

Comment:
I wonder if we should do what upstream does, `>= 10.1.0`, see https://github.com/stylelint/stylelint-config-standard/blob/master/package.json#L36